### DISCUSSION TITLE
Erreur 500 sur le page admin/report/explore ?

### DISCUSSION THREAD

**Discussion by moiiom59** [dbarzin/mercator#562] [General]
La page explore genere une erreur 500.
Merci

**Comment by dbarzin** [comment]
Pouvez-vous envoyer les logs ?

**Comment by moiiom59** [comment]
L'audit de log de Mercator ? ou des logs système et lesquells ?

**Comment by moiiom59** [comment]
Quels logs ?
La mise à jours a été faites

Bien cordialement,

**Comment by dbarzin** [comment]
Les logs de l'application dans /var/www/mercator/storage/logs/laravel.log

Pouvez-vous passer l'application en mode debug en changeant cette variable dans le fichier .env dans le répertoire /var/www/mercator

APP_DEBUG=true

Pouvez-vous reproduire l'erreur et m'envoyer une capture d'écran de la page ?

**Comment by moiiom59** [comment]
Voici
***@***.***

merci
Bien cordialement,

**Comment by dbarzin** [comment]
La capture d'écran n'est pas présente.

**Comment by moiiom59** [comment]
J’ai aussi une erreur 500 sur l’Analyse des besoins de sécurité
***@***.***

Bien cordialement,

**Comment by moiiom59** [comment]
Désolé
***@***.***

Bien cordialement,

**Comment by moiiom59** [comment]
Explore

Bien cordialement,

**Comment by moiiom59** [comment]
Bien cordialement,

**Comment by dbarzin** [comment]
Je ne vois pas les captures d'écran. Pouvez-vous les envoyer sur mon adresse gmail ?
Merci,
Didier

**Comment by dbarzin** [comment]
Cela ressemble à un problème de migration. Avez-vous bien migrer la base de données avec la commande :

    php artisan migrate

**Comment by moiiom59** [comment]
De mémoire non ! C'était une installation nouvelle.Dois je passer cette commande ?

Envoyé à partir de Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________
From: Didier Barzin ***@***.***>
Sent: Friday, December 1, 2023 6:13:41 PM
To: dbarzin/mercator ***@***.***>
Cc: moiiom59 ***@***.***>; Author ***@***.***>
Subject: [*Newsletter*] Re: [dbarzin/mercator] Erreur 500 sur le page admin/report/explore ? (Discussion #562)


Cela ressemble à un problème de migration. Avez-vous bien migrer la base de données avec la commande :

php artisan migrate

**Comment by dbarzin** [comment]
Cela fait partir de la procédure d'installation. Mais je suis très curieux de voir les logs.

**Comment by moiiom59** [comment]
Je vous ai envoyer le fichier de logs

Envoyé à partir de Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________
From: Didier Barzin ***@***.***>
Sent: Friday, December 1, 2023 6:25:45 PM
To: dbarzin/mercator ***@***.***>
Cc: moiiom59 ***@***.***>; Author ***@***.***>
Subject: [*Newsletter*] Re: [dbarzin/mercator] Erreur 500 sur le page admin/report/explore ? (Discussion #562)


Cela fait partir de la procédure d'installation. Mais je suis très curieux de voir les logs.

**Comment by dbarzin** [comment]
Le mail n'est pas arrivé.

**Comment by dbarzin** [comment]
Je viens de réinstaller Mercator dans un environnement de test. Tout fonctionne.

**Comment by moiiom59** [comment]
Que dois je faire ?

Envoyé à partir de Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by dbarzin** [comment]
Pouvez-vous m'envoyer les logs ? Rien ne m'est parvenu.
